### PR TITLE
Add Image/Video in one click

### PR DESCRIPTION
**Pull request by alex-deaconu**
Split out widget id constants
Fixed storage selector iframe component

Video Widgets are Play Until Done by default

[stage-4]

### CODE REVIEW

**Comment by alex-deaconu** [comment]
@ezequielc please review. Thanks!

File: test/e2e/editor/cases/add-content.js [hunk 1]
```diff
@@ -0,0 +1,154 @@
...
+        storageSelectorModalPage.getSelectFilesButton().click();
+        
+        helper.waitDisappear(storageSelectorModalPage.getStorageSelectorModal(), 'Storage Selector Modal');
+      });
+      
+    });
+    
+    describe('Should Add a Video item: ', function () {
+      before('Click Add Video: ', function () {
+        placeholderPlaylistPage.getAddImageButton().click();
```
Comment (ezequielc) [inline test/e2e/editor/cases/add-content.js]: @alex-deaconu Should it be `getAddVideoButton()` here? Cheers

File: test/e2e/editor/cases/add-content.js [hunk 2]
```diff
@@ -0,0 +1,154 @@
...
+
+      it('should open Widget Settings', function () {
+        helper.wait(playlistItemModalPage.getPlaylistItemModal(), 'Playlist Item Settings Page');
+        browser.switchTo().frame('widget-modal-frame');
+        
+        expect(widgetSettingsPage.getCloseButton().isDisplayed()).to.eventually.be.true;
+        expect(widgetSettingsPage.getTitle().getText()).to.eventually.equal('Text Settings');        
+      });
+
+      it('should display Playlist Item Settings Page after closing Widget Settings',function(){
```
Comment (ezequielc) [inline test/e2e/editor/cases/add-content.js]: @alex-deaconu As far as I remember the behaviour now is to not show Playlist Item Settings anymore. Do you agree? Thanks

File: test/e2e/editor/cases/add-content.js (hunk 1 above)
Comment (alex-deaconu) [inline test/e2e/editor/cases/add-content.js]: @ezequielc good catch. Fixed. Thanks.

File: test/e2e/editor/cases/add-content.js (hunk 2 above)
Comment (alex-deaconu) [inline test/e2e/editor/cases/add-content.js]: @ezequielc oops. Missed that. I need to re-work the factory a little bit, mind if I fix in a subsequent PR? Thanks.

File: web/scripts/config/prod.js [hunk 3]
```diff
@@ -21,7 +21,23 @@
...
+      // test widgets
+      'TEXT_WIDGET_PROD_OLD': 'ba0da120-7c67-437f-9caf-73585bd30c74',
+      'TEXT_WIDGET_TEST': '64cc543c-c2c6-49ab-a4e9-40ceba48a253',
+      'IMAGE_WIDGET_TEST': '2707fc05-5051-4d7b-bcde-01fafd6eaa5e',
+      'TIME_AND_DATE_WIDGET_TEST': '23e390be-8abb-4569-9084-e89722038895',
+      'RSS_WIDGET_TEST': 'b656647d-757e-448d-ab3d-b819b4244dcf',
+      'WEB_PAGE_WIDGET_TEST': 'aab933d7-ec65-499d-8c6f-e0081b8ea2ee',
+    })
+    .value('VIDEO_WIDGET', 'a7261343-1b0b-4150-a051-25d6e1b45136')
+    .value('VIDEO_WIDGET_PROD', '4bf6fb3d-1ead-4bfb-b66f-ae1fcfa3c0c6')
```
Comment (ezequielc) [inline web/scripts/config/prod.js]: @alex-deaconu Could we rename `VIDEO_WIDGET_PROD` to `VIDEO_WIDGET_TEST` to be consistent with `RENDER_WIDGETS`? Or do we need it like that for a specific reason?  Thanks!

File: test/e2e/editor/cases/add-content.js (hunk 2 above)
Comment (ezequielc) [inline test/e2e/editor/cases/add-content.js]: @alex-deaconu That would be great. Thanks

File: web/scripts/config/prod.js (hunk 3 above)
Comment (alex-deaconu) [inline web/scripts/config/prod.js]: @ezequielc renamed to `VIDEO_WIDGET_TEST`. Good catch. Thanks.

Comment (ezequielc) [inline web/scripts/config/prod.js]: @alex-deaconu Thanks!

**Comment by ezequielc** [comment]
@alex-deaconu Code looks good. 👍 Thanks!

**Pull request merged**